shell/pages/c/_cluster/apps/charts/index.vue

```diff
@@ -57,20 +67,11 @@ export default {
       debouncedSearchQuery: null,
       showDeprecated:       null,
       showHidden:           null,
-      filters:              {
-        repos:      [],
-        categories: [],
-        statuses:   [],
-        tags:       []
-      },
-      internalFilters: { // in order to update the filter checkboxes smoothly
-        repos:      [],
-        categories: [],
-        statuses:   [],
-        tags:       []
-      },
-      installedApps: [],
-      statusOptions: [
+      filters:              createInitialFilters(),
```

Comment:
we initialise filters and internalFilters with two different instances of a filter object, however in `resetAllFilters` we create one instance and that reference is set to both. feels like the latter should also have separate objects as well?